packages/charts/react-charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -379,10 +379,21 @@ export class LineChartBase extends React.Component<ILineChartProps, ILineChartSt
     return domainNRangeValue;
   };
 
-  private _injectIndexPropertyInLineChartData = (lineChartData?: ILineChartPoints[]): LineChartDataWithIndex[] | [] => {
+  private _injectIndexPropertyInLineChartData = (
+    lineChartData?: ILineChartPoints[],
+    isFilterSelectedLegends?: boolean,
```

Comment:
make this as default = false. you wont need to change the existing code.

Reply:
oh yes.